source/jormungandr/jormungandr/schedule.py

```diff
@@ -303,6 +303,9 @@ def comparator(p1, p2):
             return cmp(p1.stop_date_time.departure_date_time, p2.stop_date_time.departure_date_time)
 
         resp.next_departures.sort(comparator)
+        count = request['count']
+        if len(resp.next_departures) > count and rt_proxy:
```

Comment:
Are you sure you don't want to trim the next departures event if we have no `rt_proxy` ?

Reply:
it's not required as kraken already return at most `count` items, but in fact I added this test because something else was failing, forgot to remove it